packages/sanes-chrome-extension/src/extension/background/model/persona/index.ts

```diff
@@ -136,6 +137,7 @@ export class Persona {
       signer: Encoding.toHex(t.transaction.creator.pubkey.data),
       memo: t.transaction.memo,
       amount: t.transaction.amount,
+      blockExplorerUrl: null,
```

Comment:
Is there a different ticket to set this?

Reply:
Yes my next PR :) which will be done tomorrow for closing #265 (wrote in the description)

Reply:
Ah, sorry, I only read the list items